### PR TITLE
Refs #19332: Fix typo "smart proy"

### PR DESCRIPTION
**Pull request by akofink**
This message only shows if you have no smart proxies.

### CODE REVIEW

File: app/views/smart_proxies/welcome.html.erb [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <%= icon_text("sitemap", "", :kind => "fa") %>
   </div>
   <h1><%= _('Smart Proxies') %></h1>
-  <p><%= _("Smart proy provides restful API to subsystems such as DNS, DHCP, TFTP that Foreman talks to") %></p>
+  <p><%= _("Smart proxy provides restful API to subsystems such as DNS, DHCP, TFTP that Foreman talks to") %></p>
```
Comment (stbenjam) [inline app/views/smart_proxies/welcome.html.erb]: "Smart proxies provide" sounds more natural, but I don't know why.

Comment (akofink) [reply app/views/smart_proxies/welcome.html.erb]: It also ends in a preposition, which is a no no.

Comment (stbenjam) [reply app/views/smart_proxies/welcome.html.erb]: How about just "Smart proxies provide a RESTful API to manage subsystems like DNS, DHCP, and TFTP." ?

**Comment by akofink** [comment]
Updated

**Review by stbenjam — APPROVED** [review]

**Comment by sean797** [comment]
> In fact in last upstream discussion I believe it was agreed that Foreman Proxy is the preferred name[1]. While we still have Smart Proxy all around, I think we should not add more
> 
> [1] https://groups.google.com/forum/#!searchin/foreman-dev/smart$20or$20foreman$20proxy$20name%7Csort:relevance/foreman-dev/H-jbnDY5mhI/qHofd_dvBAAJ

from https://github.com/theforeman/foreman-discovery-image/pull/98#issuecomment-336797420

**Comment by akofink** [comment]
@sean797 I could change this and all other (5 total) occurrences of "Smart Proxy" in this view, change just this one and leave the file inconsistent, or leave it as is and fix only what the issue states. I don't like leaving the file inconsistent, of course; however, I also can't change all 5 occurrences (one is a documentation url) without changing other files. I think the best thing to do is to wait for an issue that does s/Smart Proxy/Foreman Proxy/ in one PR. :man_shrugging: What do you think?

**Review by ekohl — APPROVED** [review]
This does sound like an improvement already and is consistent with the rest of the section so :+1:

**Comment by mmoll** [comment]
test failures unrelated.

**Comment by sean797** [comment]
Yes - I agree @akofink & @ekohl :+1:

**Comment by akofink** [comment]
[test]

**Comment by akofink** [comment]
[test katello]

**Comment by akofink** [comment]
[test foreman]

**Review by dLobatog — APPROVED** [review]

**Pull request merged**